app/views/new_threads/show.html.erb

```diff
@@ -1,89 +1,79 @@
-<% @pageTitle = @new_thread.title + " | CAT Forum" %>
+<!-- <%= @pageTitle = @new_thread.title %>
 <% @PageDescription = @new_thread.description %>
-<% if user_signed_in? %>
-  <span class="user"> <%= current_user.username %></span>
-  <%= image_tag current_user.image.url(:thumb), size: "50x50", :class => 'user-img'%>
-<% end %>                                            
-<div class="thread-replies" itemscope itemtype="http://schema.org/Question">
-  <%= image_tag(@new_thread.user.image.url(:thumb), :class => "user-image") %>
-  <%= image_tag("arrow.png", class: "triangle") %>
-  <div class="speech-bubble">
+-->
```

Comment:
>  actually without the comment the title is printed there

It is because you have used `<%= %>` instead of `<% %>`. Former is will display the content in html page while later just execute the code but not print anything. 

Check out this question: http://stackoverflow.com/questions/7996695/what-is-the-difference-between-and-in-erb-in-rails